scripts/concat.js

```diff
@@ -23,6 +23,15 @@ shell.cat([
   'src/flavors/' + flavor + '/static/css/custom.css'
 ]).to('src/base/static/dist/cat-bundle.css')
 
+shell.cat([
+  'src/base/static/libs/binaryajax.js',
+  'src/base/static/libs/exif.js',
+  'src/base/static/libs/swag.min.js',
+  'src/base/static/libs/leaflet.sidebar.js',
+  'src/base/static/libs/leaflet-wmts.js',
+  'src/base/static/libs/leaflet.argo.js'
+]).to('src/base/static/dist/cat-lib-bundle.js')
+
```

Comment:
Duplicating the paths to our libs like this isn't ideal and prone to bugs. I think it would be better to bundle our js resources using webpack. Or even better - refactor these libs into separate modules that are imported into our codebase via npm, but I understand that's an ongoing process.

Eventually, I would also like to remove this whole file, along with our `node-sass` build step, by bundling our CSS using webpack as well. Here's a webpack plugin that I'm looking to use for that: https://www.npmjs.com/package/extract-text-webpack-plugin